### PR TITLE
Implemented server skeleton, basic db connection to postgresql

### PR DESCRIPTION
**Pull request by Taylor-Sheets3**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -15,5 +16,12 @@
   "bugs": {
     "url": "https://github.com/blueOceanOne/PuppyMates/issues"
   },
-  "homepage": "https://github.com/blueOceanOne/PuppyMates#readme"
+  "homepage": "https://github.com/blueOceanOne/PuppyMates#readme",
+  "dependencies": {
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "nodemon": "^2.0.20",
```
Comment (KateFeaster) [inline backend/package.json:23]: Nodemon should be dev dependency

File: backend/server/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require('dotenv').config();
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, '../../frontend/index.js')));
```
Comment (boyiq) [inline backend/server/index.js:7]: Just wanted to confirm if we are serving this index.js at the frontend.

Comment (KateFeaster) [inline backend/server/index.js:7]: Remove static serving

**Review by KateFeaster — CHANGES_REQUESTED** [review]
Code looks fantastic! Added a couple of changes and then it should be gtg!

**Review by boyiq — APPROVED** [review]
Everything else looks good to me except the comment

File: backend/package.json (hunk 1 above)
Comment (Taylor-Sheets3) [reply backend/package.json]: Nodemon is now a backend dev dependency

File: backend/server/index.js (hunk 2 above)
Comment (Taylor-Sheets3) [reply backend/server/index.js]: removed

**Pull request merged**